app/controllers/settings_controller.rb

```diff
@@ -22,4 +23,15 @@ def update
       render :json => {"errors" => error_msg}, :status => :unprocessable_entity
     end
   end
+
+  private
+
+  # reload settings when code reloads after change in development env
+  def reload_settings
+    Setting.descendants.each(&:load_defaults)
```

Comment:
`load_defaults` method run only when foreman initialize, but not  when development env reload after code change.